scripts/ableplayer-base.js

```diff
@@ -551,3 +557,9 @@ var AblePlayerInstances = [];
 	AblePlayer.youtubeIframeAPIReady = false;
 	AblePlayer.loadingYoutubeIframeAPI = false;
 })(jQuery);
+
+// Exports the creating of AblePlayer instances, which
+// needs to be called after a react component is mounted
+exports.getAllAblePlayers = window.getAllAblePlayers;
+//  Exports the instances created, for debugging purposes
+exports.ablePlayerInstances = AblePlayerInstances;
```

Comment:
Does `ablePlayerInstances` get be updated when you run `getAllAblePlayers` on the client code?

Reply:
Yes :)